samples/hybrid-connections/dotnet/hcreverseproxy/Microsoft.Azure.Relay.ReverseProxy/HybridConnectionReverseProxy.cs

```diff
@@ -75,6 +75,16 @@ async Task SendResponseAsync(RelayedHttpListenerContext context, HttpResponseMes
 
                 context.Response.Headers.Add(header.Key, string.Join(",", header.Value));
             }
+            
+            foreach (KeyValuePair<string, IEnumerable<string>> header in responseMessage.Content.Headers)
+            {
+                if (string.Equals(header.Key, "Transfer-Encoding"))
```

Comment:
>"Transfer-Encoding" [](http://example.com/codeflow?start=46&length=19)

In your testing was Transfer-Encoding ever observed here?

Reply:
[HttpContentHeaders static constructor](https://github.com/microsoft/referencesource/blob/5697c29004a34d80acdaf5742d7e699022c64ecd/System/net/System/Net/Http/Headers/HttpContentHeaders.cs#L171) calls 
[HttpGeneralHeaders.AddKnownHeaders](https://github.com/microsoft/referencesource/blob/5697c29004a34d80acdaf5742d7e699022c64ecd/System/net/System/Net/Http/Headers/HttpGeneralHeaders.cs#L221) which puts TransferEncoding into that invalidHeaders list.

Reply:
If it works in your testing will you simply remove the if statement here?